source/draw/gpu/GpuCanvasYuv420Semiplanar.ooc

```diff
@@ -76,9 +76,10 @@ GpuCanvasYuv420Semiplanar: class extends GpuCanvas {
 		uvLines free()
 	}
 	drawPoints: override func (pointList: VectorList<FloatPoint2D>) { this _target y canvas drawPoints(pointList) }
-	fill: override func {
-		this _target y canvas fill()
-		this _target uv canvas fill()
+	fill: override func ~color (color: ColorRgba) {
+		yuv := color toYuv()
+		this _target y canvas fill(ColorRgba new(yuv y, 0, 0, 255))
+		this _target uv canvas fill(ColorRgba new(yuv u, yuv v, 0, 255))
```

Comment:
It is copied from the Pen setter to maintain full backward compatibility. Y and UV are stored in RGBA just like on the GPU instead of being converted from RGBA on the planar level because a monochrome could just as well be used to represent a planar RGB image or an intensity map. It is an ugly solution but I don't see a better way since each packed image knows nothing about the planar image.
```ooc
 pen: Pen {
 	get { this _pen }
	set(value) {
		this _pen = value
 		yuv := value color toYuv()
 		this _target y canvas pen = Pen new(ColorRgba new(yuv y, 0, 0, 255), value width)
 		this _target uv canvas pen = Pen new(ColorRgba new(yuv u, yuv v, 0, 255), value width)
	}
}
```